cogs/coding.py

```diff
@@ -47,7 +45,9 @@ def get_h2_content(self, tag: NavigableString) -> str:
                 continue
             if element.name == "h2":
                 break
-            siblings.append(element.text)  # get the text of the elemets before the next h2
+            siblings.append(
+                element.text
+            )  # get the text of the elemets before the next h2
```

Comment:
I'm not a huge fan of these splits, they aren't that big and in many cases, it's split only because the comment is long, it'd be wiser to simply move the comment before the action instead of splitting short functions with only 1 argument.